apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -1807,7 +1809,6 @@ class LivechatClass {
 				LivechatInquiry.setNameByRoomId(rid, name),
 				Subscriptions.updateDisplayNameByRoomId(rid, name),
 			]);
-			return true;
```

Comment:
Needed?

Reply:
yes! otherwise `notifyOnRoomChangedById` on line 1813 will not be called, which is the whole point =P

Reply:
Ahh, the thing outside the if, got it